internal/route/routes.go

```diff
@@ -17,8 +17,10 @@ import (
 // HandleIndex is the route for "/"
 func HandleIndex(w http.ResponseWriter, r *http.Request) {
 	tpl := template.Must(template.ParseFiles("web/template/index.html"))
-	var todo string
-	var name string
+	var tplvars struct {
+		Name string
+		Todo string
+	}
```

Comment:
This is called a struct literal. Perfect for HTML templates. We don't predefine a type we just say `var <name> struct { ... fields }`

Now we can use "tplvars" wherever in this func and pass it without that weird structure we had below.